### DISCUSSION TITLE
Skin editor doesn't account for beatmap's ruleset when editing gameplay

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#22259] [Q&A] [answered]
Choose non-osu! map, exit to menu, choose another ruleset, enter editor and click "gameplay".

some pairs works okay, just looks a little weird.

https://user-images.githubusercontent.com/53872073/212946508-41dda563-3dc6-4a14-824e-fa5e0ae99c44.mp4

something -> catch fails as usual:

https://user-images.githubusercontent.com/53872073/212946641-3f5f4ee8-a995-46f3-a4a3-5969f0cc7e1e.mp4

[runtime.log](https://github.com/ppy/osu/files/10436857/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22263
